pkg/ottl/contexts/internal/value.go

```diff
@@ -71,27 +71,35 @@ func SetValue(value pcommon.Value, val any) error {
 func getIndexableValue[K any](ctx context.Context, tCtx K, value pcommon.Value, keys []ottl.Key[K]) (any, error) {
 	val := value
 	var ok bool
-	for i := 0; i < len(keys); i++ {
+	for count := 0; count < len(keys); count++ {
```

Comment:
What does `count` represent here? To me the conventional `i` makes more sense, but I could be missing your intent. `count` strikes me as more of an accumulated/computed value.